plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -78,14 +78,6 @@
               artifactId="servlet-api"
               version="2.5" />
       </libraryFile>
-    </library>
-
-    <library
```

Comment:
My recollection is that JSPs didn't work locally until we added this.

Reply:
I haven't removed it: I've combined it into the `appengine-servlet-2.5 ` container.